package.json

```diff
@@ -30,13 +30,14 @@
   "devDependencies": {
     "browser-sync": "^2.14.0",
     "codelyzer": "0.0.28",
+    "del": "^2.2.2",
     "gulp": "^3.9.1",
     "gulp-inject": "^4.1.0",
     "gulp-sourcemaps": "^1.6.0",
     "gulp-typescript": "^2.13.6",
-    "gulp-uglify": "^2.0.0",
```

Comment:
Yeah, I stripped it right back so that we can get a good base script, to which we can add features like uglify/minify/bundling scripts etc.